syntax = "proto3";

import 'google/protobuf/empty.proto';

package samples.hello.cv_service.messages;

// Union of all possible commands of this package.
message Command {
    oneof id {
        google.protobuf.Empty processing_start = 1;
        google.protobuf.Empty processing_stop = 2;
    }
}

// Union of all possible events of this package.
message Event {
    oneof id {
        google.protobuf.Empty close = 1;
        google.protobuf.Empty far = 2;
        float depth_mean = 3;
    }
}
